dbt-adapters/src/dbt/adapters/sql/impl.py

```diff
@@ -255,6 +257,10 @@ def validate_sql(self, sql: str) -> AdapterResponse:
         return adapter_response
 
     # This is for use in the test suite
+    @available
```

Comment:
What's the rationale for making this available? From the comment, I struggle to see why this method even exists on this class, and isn't instead implemented on the test fixture.

Reply:
@colin-rogers-dbt added this one, and I'm not sure it's strictly necessary?